### DISCUSSION TITLE
3D Model is not starting at all, only the poster is shown

### DISCUSSION THREAD

**Discussion by rotgrau** [google/model-viewer#3416] [Q&A]
I just click "Download scene" for any model from the editor, like the astronaut and then open the index.html, all that happens is that a progress bar flashes and the poster.webp is shown. I did not change any of the default settings in the editor.
I'm working on a new MacBook Pro M1, tried all Browsers.

**Comment by elalish** [comment]
Unfortunately there isn't a progress bar for the sometimes long task of exporting a model, which is what you're waiting for. Also, unlike model-viewer, the editor is only well-tested under Chrome. Try a smaller model and being patient; also check the JS console logs for errors/warnings. It's working on my M1, so hopefully this helps. Please let us know any further details you notice.
